source/push-to-deploy.md

```diff
@@ -0,0 +1,71 @@
+---
+title: Push To Deploy
+order: 19
+description: Learn how to setup Push to Deploy on Cloud, so you can deploy changes to your apps just buy by performing a commit to a repository.
+---
+
+With this feature, never was easier to create an app on Galaxy. All you need to do is follow three  simple steps, then you'll have a new app deployed.
+
+<h2 id="pre-requirements">Pre requirements</h2>
+
+Today we have our integration just with GitHub. In the future we'll add more Git providers.
+
+The only requirement you need today to start using this feature is to have a repository on GitHub with a Meteor project on it. And that's it.
+
+<h2 id="deploying-a-new-app">Deploying a new app</h2>
+
+Start by going to your Cloud account [here](https://cloud.meteor.com/). Then go to *apps* > *Deploy a new app*. Once there, you'll see this page:
+
+<img src="images/push-to-deploy-first-step.png" style="width: 780px;">
+
+Go ahead and link your GitHub account to our app, just by clicking on the GitHub button.
```

Comment:
There is also the authorization step.

And I believe if they want to authorize more apps later, at least today, they need to change permissions on Github.

Reply:
@denihs ?

Reply:
:+1: